src/main/kotlin/no/nav/familie/ef/sak/journalføring/JournalføringController.kt

```diff
@@ -43,6 +46,21 @@ class JournalføringController(private val journalføringService: Journalføring
         return Ressurs.success(journalføringService.hentDokument(journalpostId, dokumentInfoId))
     }
 
+    @GetMapping("/{journalpostId}/dokument-pdf/{dokumentInfoId}")
+    fun hentDokumentSomPdf(@PathVariable journalpostId: String, @PathVariable dokumentInfoId: String): ResponseEntity<ByteArray> {
+        val (_, personIdent) = finnJournalpostOgPersonIdent(journalpostId)
+        tilgangService.validerTilgangTilPersonMedBarn(personIdent)
+        val responseHeaders = HttpHeaders()
+        responseHeaders.contentType = MediaType.valueOf("application/pdf")
+        responseHeaders.add("Content-Disposition", "inline; filename=" + "dokument.pdf")
+
+        return ResponseEntity(
+                journalføringService.hentDokument(journalpostId, dokumentInfoId),
+                responseHeaders,
+                HttpStatus.OK
+        )
```

Comment:
Hvis dokumentanvnet ikke er viktig burde det vel holde med: 
val (_, personIdent) = finnJournalpostOgPersonIdent(journalpostId)
tilgangService.validerTilgangTilPersonMedBarn(personIdent)
return journalføringService.hentDokument(journalpostId, dokumentInfoId),

Reply:
Trodde vi trengte `inline` for å at chrome ikke skulle prøve å laste ned, men ser ut som du har rett 😂